docs/lake/README.md

```diff
@@ -226,7 +236,6 @@ zed lake load -i parquet sample4.parquet
 zed lake load -i zst sample5.zst
 ```
 
-By default, the data is committed into the `main` branch of the pool.
```

Comment:
I deleted this line because it doesn't seem accurate anymore. Yes, if someone had done only `zed lake use -p pool` it's true that data would by default be committed into the `main` branch of that pool. But in the absence of _any_ `zed lake use`, the `load` would simply fail.

```
$ zed lake init
using environment variable ZED_LAKE_ROOT
lake created: /Users/phil/logs

$ zed lake create logs
pool created: logs 1xpaHDcvgYOOWBWiJTwPxbZTWLP

$ zed lake load dpd.log.gz 
HEAD not specified: indicate with -use or run the "use" command
```

There's text a little further down in the same section that already says:

> Or, as mentioned above, you can set the default branch for the load command via `use`: ...

Therefore I think we're covered without the line I'm deleting here.